api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,232 @@
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (41, 2, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (42, 3, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (43, 4, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (44, 5, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (45, 6, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (46, 7, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (47, 8, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (48, 9, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (49, 10, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (50, 11, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (51, 12, 'H');
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, CARD_SUIT_ID) VALUES (52, 13, 'H');
+
+CREATE TABLE ROUNDS (
+    ROUND_ID VARCHAR2(255 BYTE) NOT NULL,
+    ROUND VARCHAR2(255 BYTE) NOT NULL,
+    PRIMARY KEY (ROUND_ID)
+);
+
+COMMENT ON TABLE ROUNDS IS 'The rounds that are possible to appear in a game';
+COMMENT ON COLUMN ROUNDS.ROUND_ID IS 'The ID of the round';
+COMMENT ON COLUMN ROUNDS.ROUND IS 'The name of the round';
+
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES ('B', 'blind');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES ('F', 'flop');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES ('T', 'turn');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES ('R', 'river');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES ('S', 'showdown');
+
+CREATE TABLE MEMBERS (
+    MEMBER_ID NUMBER NOT NULL,
+    MEMBER_NICKNAME VARCHAR2(255 BYTE) NOT NULL,
+    MEMBER_EMAIL VARCHAR2(255 BYTE) NOT NULL,
+    MEMBER_LEVEL NUMBER NOT NULL,
+    MEMBER_EXP_OVER_LEVEL NUMBER NOT NULL,
+    MEMBER_PASSWORD VARCHAR2(255 BYTE) NOT NULL,
+    PRIMARY KEY (MEMBER_ID),
+    UNIQUE (MEMBER_EMAIL),
+    UNIQUE (MEMBER_NICKNAME)
+);
+
+COMMENT ON TABLE MEMBERS IS 'The member in the system';
+COMMENT ON COLUMN MEMBERS.MEMBER_ID IS 'The ID of the member';
+COMMENT ON COLUMN MEMBERS.MEMBER_NICKNAME IS 'The user name of the member';
+COMMENT ON COLUMN MEMBERS.MEMBER_EMAIL IS 'The email address of the member';
+COMMENT ON COLUMN MEMBERS.MEMBER_LEVEL IS 'The level of member in the system';
+COMMENT ON COLUMN MEMBERS.MEMBER_EXP_OVER_LEVEL IS 'The experience of member that is above the current level';
+COMMENT ON COLUMN MEMBERS.PASSWORD IS 'The password of member, which should be stored in hashed form';
+
+INSERT INTO MEMBERS (MEMBER_ID, MEMBER_NICKNAME, MEMBER_EMAIL, MEMBER_LEVEL, MEMBER_EXP_OVER_LEVEL, MEMBER_PASSWORD) VALUES (1, 'guz', 'guz@oregonstate.edu', 123, 10492, 'hunter3');
+INSERT INTO MEMBERS (MEMBER_ID, MEMBER_NICKNAME, MEMBER_EMAIL, MEMBER_LEVEL, MEMBER_EXP_OVER_LEVEL, MEMBER_PASSWORD) VALUES (2, 'John Wick', 'wickj@oregonstate.edu', 200, 114514, 'hunter10');
```

Comment:
Also since this looks like it's just test data, could you add it to the bottom as a SQL comment or remove it from the script?

Reply:
I will remove it after the endpoints are implemented.